src/components/AppHeader.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class='app-header'>
+    <div class='ui secondary pointing menu'>
+      <router-link to='/' class='item'>Feed</router-link>
```

Comment:
I'd like to see another view that is basically like your `following` but as a feed.... Maybe that's what you intended for `following` to be.